src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-7-0-319.mdx

```diff
@@ -9,6 +9,15 @@ security: []
 ---
 ## New Relic PHP Agent v10.7.0.319
 
+### Important note
+
+If the following error occurs when installing the newrelic-php5 Debian package:
+
+```
+Setting up newrelic-php5 (10.7.0.319) ...
+Unknown PHP version: 8.2
+```
```

Comment:
@mfulb Is this okay?

Reply:
Sure - lets see how it looks on preview - thanks for working through this with me.

Reply:
It seems to have formatted those two lines like I want but didn't show the remaining text in that section.

Reply:
Ah yes, accidentally deleted that line. It's back now.